CONTRIBUTING.md

```diff
@@ -60,7 +60,47 @@ We use changesets to automatically version and publish releases, and generate re
 
 ### Core SDK
 
-The CLI depends on the [SmartThings Core SDK](https://github.com/SmartThingsCommunity/smartthings-core-sdk). To use a pre-release version of the SDK for testing purposes, you'll need to make any required changes in both checked out repositories and then [npm-link](https://docs.npmjs.com/cli/v8/commands/npm-link#workspace-usage) the SDK into the CLI.
+The CLI depends on the [SmartThings Core SDK](https://github.com/SmartThingsCommunity/smartthings-core-sdk). To use a pre-release version of the SDK for testing purposes, you'll need to make any required changes in both checked-out repositories and then [npm-link](https://docs.npmjs.com/cli/v8/commands/npm-link#workspace-usage) the SDK into the CLI.
+
+### Command Documentation
+
+The SmartThings CLI is built using [oclif](https://oclif.io), which allows for self-documenting
+commands. Some ground rules to follow when creating or updating commands:
+
+* Commands should be kebab case. The module name is used as the command name and can be placed in
+  a topic by placing it in that topic's directory.
+* Commands should always include a short one-line description that starts with a lowercase letter
+  and does not end with a period. A more detailed description can be included after a newline. This
+  description can be split across lines but should not include newlines (oclif will wrap it).
+* Flags are also kebab case and should have a short single-character version only if they ar
+  expected to be used often.
+  * Descriptions of flags should be short, start with a lowercase letter, and include no ending
+    punctuation.
+  * Flags for ids should not include an `-id` suffix.
+* Names for arguments should be camel case starting with a lowercase letter.
+  * Descriptions for arguments should be a short phrase without a capital letter and without ending
+    punctuation.
+* Examples of common tasks should be included with a description of the example.
+  * Use the [object example format](https://oclif.io/docs/commands#other-command-options).
+    Do not use the "simple string" format since it produces different output.
```

Comment:
It took me a moment to key in on what the "simple string" format was when I followed the link. It might be more clear to say something like "Use the object format rather than the simple string format when providing examples". I think it's more direct to just leave it at that, but if it's important to clarify why to use one and not the other, you could append ", as they produce different output."

Reply:
I somehow missed this the first time I looked at it. I've fixed this wording now.